### PR TITLE
Adds the article "Data Orchestration Tools Compared: Why Kestra Stands Out for Data Engineers"

### PR DESCRIPTION
**Pull request by federico-trotta**
The main idea is to repropose the same schema of the latest articles but with a focus on data here.

For this reason, the tools present a 'best for' section in this case.

Also, I've strived to not repeat myself, in particular referencing it to [this one](https://github.com/kestra-io/docs/pull/1813). This is why, for example, I mentioned even here the high available scheduler, but in this case I focused on data.

Let me know how we can improve it. :)

### CODE REVIEW

**Review by wrussell1999 — CHANGES_REQUESTED** [review]
I've given this a quick once through and left some comments. I'll give it a more in-depth read once you've addressed the initial feedback!

File: content/blogs/2024-MM-DD-comparing-data-orchestrators.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+- **Scalability needs**: If your data workflows require horizontal scalability to handle large datasets or fluctuating workloads, Kestra provides robust scalability out of the box. This makes it a strong choice for data engineers working in environments with complex data pipelines that need to scale efficiently.
+- **Quick onboarding**: With Kestra’s intuitive interface and workflow definitions, data teams can onboard faster and start orchestrating their data pipelines without steep learning curves. This is particularly useful for teams that need to operationalize their data workflows quickly in fast-paced environments.
+
+### When to Choose Kestra Over Prefect
+- **Cost efficiency**: In data-driven organizations that prioritize budget control, Kestra’s fully open-source nature eliminates the need for paid tiers, making it a more cost-effective solution than Prefect’s cloud offerings. This can be critical for data teams that want to optimize orchestration without incurring additional software costs.
```
[the remark this answers is not in the document]
Comment (federico-trotta) [reply content/blogs/2024-MM-DD-comparing-data-orchestrators.md:167]: My understanding is that Prefect is not fully open source. It provides a "core" (open-source) and a "cloud version" (paid). In particular, the [cloud version](https://www.restack.io/docs/prefect-knowledge-prefect-open-source-vs-cloud) provides some features more than the core one. For example, if we consider that the cloud version provides "automation" and "email notifications" we can say that Kestra already provides it, and it's fully open-source, thus cost-efficiency.

Another resource is on their [website](https://www.prefect.io/cloud-vs-oss). Here we can see that the cloud version offers webhooks and system metrics...

Let me know if I'm getting this wrong... :)

[the remark this answers is not in the document]
Comment (wrussell1999) [reply content/blogs/2024-MM-DD-comparing-data-orchestrators.md:166]: Ah gotcha. I still think that we're in a similar situation as them where we have an enterprise edition that has a bunch of non open source features too. Instead you could word it that we allow you to host on prem for both offerings, and not limited to the cloud @federico-trotta

[the remark this answers is not in the document]
Comment (federico-trotta) [reply content/blogs/2024-MM-DD-comparing-data-orchestrators.md]: how about that @wrussell1999 ? :)

File: content/blogs/2024-MM-DD-comparing-data-orchestrators.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+In this article, we describe what data orchestration is, how to choose the right data orchestration tool, and compare some of the most popular options: Apache Airflow, Prefect, Dagster, and Kestra.
+
+## What Is Data Orchestration?
+Data [orchestration](./2024-09-18-what-is-an-orchestrator.md) is an automated process that combines and organizes data from different data sources. You can think of it like being the conductor of a massive data symphony that involves the automated coordination and management of data as it moves between different systems and processes. Imagine having to manually manage every data pipeline, handle dependencies, schedule tasks, and monitor workflows: that would be a nightmare, right? Well, data orchestration tools take care of all that heavy lifting for you.
```
Comment (wrussell1999) [inline content/blogs/2024-MM-DD-comparing-data-orchestrators.md:17]:
```suggestion
Data [orchestration](./2024-09-18-what-is-an-orchestrator.md) is orchestration of workflow nodes that touch data. You can think of it like being the conductor of a massive data symphony that involves the automated coordination and management of data as it moves between different systems and processes. Imagine having to manually manage every data pipeline, handle dependencies, schedule tasks, and monitor workflows: that would be a nightmare, right? Well, data orchestration tools take care of all that heavy lifting for you.

```

File: content/blogs/2024-MM-DD-comparing-data-orchestrators.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (anna-geller) [reply content/blogs/2024-MM-DD-comparing-data-orchestrators.md]: Federico, when comparing OSS tools, you should check the LICENSE in each repo. Prefect is fully open source with Apache 2.0 license, same as kestra and dagster

[the remark this answers is not in the document]
Comment (federico-trotta) [reply content/blogs/2024-MM-DD-comparing-data-orchestrators.md]: thank you! I recently came across the concept of "source-available" which, possibly, led me to some confusion.

**Pull request closed without merging**